src/status_im/contexts/wallet/edit_account/view.cljs

```diff
@@ -35,34 +36,34 @@
 
 (defn view
   []
-  (let [edited-account-name  (reagent/atom nil)
-        show-confirm-button? (reagent/atom false)
-        on-change-color      (fn [edited-color {:keys [color] :as account}]
-                               (when (not= edited-color color)
-                                 (save-account {:account     account
-                                                :updated-key :color
-                                                :new-value   edited-color})))
-        on-change-emoji      (fn [edited-emoji {:keys [emoji] :as account}]
-                               (when (not= edited-emoji emoji)
-                                 (save-account {:account     account
-                                                :updated-key :emoji
-                                                :new-value   edited-emoji})))
-        on-confirm-name      (fn [account]
-                               (rn/dismiss-keyboard!)
-                               (save-account {:account     account
-                                              :updated-key :name
-                                              :new-value   @edited-account-name}))]
+  (let [edited-account-name (reagent/atom nil)
+        on-change-color     (fn [edited-color {:keys [color] :as account}]
+                              (when (not= edited-color color)
+                                (save-account {:account     account
+                                               :updated-key :color
+                                               :new-value   edited-color})))
+        on-change-emoji     (fn [edited-emoji {:keys [emoji] :as account}]
+                              (when (not= edited-emoji emoji)
+                                (save-account {:account     account
+                                               :updated-key :emoji
+                                               :new-value   edited-emoji})))
+        on-confirm-name     (fn [account]
+                              (rn/dismiss-keyboard!)
+                              (save-account {:account     account
+                                             :updated-key :name
+                                             :new-value   @edited-account-name}))]
     (fn []
       (let [{:keys [name emoji address color watch-only? default-account?]
```

Comment:
nit: we can destructure name as wallet-name to avoid shadowing `clojure.core/name`